dipy/direction/probabilistic_direction_getter.pyx

```diff
@@ -32,8 +33,7 @@ cdef class ProbabilisticDirectionGetter(PmfGenDirectionGetter):
         double[:, :] vertices
         dict _adj_matrix
 
-    def __init__(self, pmf_gen, max_angle, sphere=None, pmf_threshold=0.1,
-                 **kwargs):
+    def __init__(self, pmf_gen, max_angle, sphere, pmf_threshold=.1, **kwargs):
```

Comment:
is the `sphere` mandatory in all `DirectionGetter` ?

Reply:
for all `BasePmfDirectionGetter` yes. There is a default value in the `PmfGenDirectionGetter.from_pmf(.)` and `PmfGenDirectionGetter.from.shcoefs(.)`. If sphere is None, the directionGetters won't work.

Reply:
make sense, thanks!